test/utility/test_util.h

```diff
@@ -84,6 +85,11 @@ typedef struct {
     uint64_t data_seed;      /* Random seed for data ops */
     uint64_t extra_seed;     /* Random seed for extra ops */
 
+    uint32_t force_delay; /* Force a simulated network delay every N operations */
+    uint32_t delay_ms;    /* Average length of delay when simulated */
+    uint32_t force_error; /* Force a simulated network error every N operations */
+    uint32_t error_ms;    /* Average length of delay when simulated */
```

Comment:
Alphabetize all these.

Reply:
Fixed in 0e6b43406c158379072274241f2c15de26c317b5